src/test/java/io/github/perrymant/moneymaker/CliTest.java

```diff
@@ -80,14 +86,14 @@ public void givenHelpLogsHelpMessage() {
 
     @Test
     public void givenNoArgsLogsErrorMessage() {
```

Comment:
This method name is no longer accurate.

```suggestion
    public void givenGarbageArgsLogsErrorMessage() {
```